languages/javalang/builder-renderer/java-builder-client/src/main/kotlin/io/vrap/codegen/languages/javalang/client/builder/requests/JavaHttpRequestRenderer.kt

```diff
@@ -70,59 +70,59 @@ class JavaHttpRequestRenderer @Inject constructor(override val vrapTypeProvider:
             |}
         """.trimMargin()
                 .keepIndentation()
-
+        
         return TemplateFile(
                 relativePath = "${vrapType.`package`}.${type.toRequestName()}".replace(".", "/") + ".java",
                 content = content
         )
     }
-
+    
     private fun Method.constructor(): String? {
         val constructorArguments = mutableListOf("final ApiHttpClient apiHttpClient")
         val constructorAssignments = mutableListOf("this.apiHttpClient = apiHttpClient;")
 
         this.pathArguments().map { "String $it" }.forEach { constructorArguments.add(it) }
         this.pathArguments().map { "this.$it = $it;" }.forEach { constructorAssignments.add(it) }
 
-        if (this.bodies != null && this.bodies.isNotEmpty()) {
-            if (this.bodies[0].type.toVrapType() is VrapObjectType) {
+        if(this.bodies != null && this.bodies.isNotEmpty()){
+            if(this.bodies[0].type.toVrapType() is VrapObjectType) {
                 val methodBodyVrapType = this.bodies[0].type.toVrapType() as VrapObjectType
                 val methodBodyArgument = "${methodBodyVrapType.`package`.toJavaPackage()}.${methodBodyVrapType.simpleClassName} ${methodBodyVrapType.simpleClassName.decapitalize()}"
                 constructorArguments.add(methodBodyArgument)
                 val methodBodyAssignment = "this.${methodBodyVrapType.simpleClassName.decapitalize()} = ${methodBodyVrapType.simpleClassName.decapitalize()};"
                 constructorAssignments.add(methodBodyAssignment)
-            } else {
+            }else {
                 constructorArguments.add("com.fasterxml.jackson.databind.JsonNode jsonNode")
                 constructorAssignments.add("this.jsonNode = jsonNode;")
             }
         }
-
+        
         return """
             |public ${this.toRequestName()}(${constructorArguments.joinToString(separator = ", ")}){
             |   <${constructorAssignments.joinToString(separator = "\n")}>
             |}
         """.trimMargin().keepIndentation()
     }
-
+    
     private fun Method.fields(): String? {
 
-        val parameterFields: String =
+        val parameterFields : String =
                 this.queryParameters
-                        .filter { it.annotations.find { it.type.name.equals(PLACEHOLDER_PARAM_ANNOTATION) } == null }
+                        .filter { it.annotations.find { it.type.name.equals(PLACEHOLDER_PARAM_ANNOTATION) } == null}
```

Comment:
What does the `PLACEHOLDER_PARAM_ANNOTATION` do? Can we somehow explain it a bit in the code? It#s really confusing that the constants name contains `ANNOTATION` but then we try to find a type with this name...

Reply:
@jenschude can answer this I think. I dont know what it is. Took it from php gen.